pandas/core/tools/numeric.py

```diff
@@ -211,8 +211,15 @@ def to_numeric(
 
     values_dtype = getattr(values, "dtype", None)
     if isinstance(values_dtype, ArrowDtype):
+        if is_numeric_dtype(values_dtype):
+            if is_series:
+                return arg._constructor(values, index=arg.index, name=arg.name)
+            else:
+                return values
```

Comment:
Could you try to specify `if not is_numeric_dtype` here, and let the logic below try to convert this case anyways? This short circuit would skip some of the options set in `to_numeric`

Reply:
Thanks @simonjayhawkins , I already create anothter PR for document issue: https://github.com/pandas-dev/pandas/pull/61904